frontends/slack/src/main/kotlin/com/gatehill/corebot/chat/SlackSessionServiceImpl.kt

```diff
@@ -42,8 +43,18 @@ open class SlackSessionServiceImpl @Inject constructor(configService: ConfigServ
     override val botUsername: String
         get() = session.sessionPersona().userName
 
-    override fun sendMessage(channelId: String, message: String) {
-        session.sendMessage(session.findChannelById(channelId), message)
+    override fun sendMessage(channelId: String, threadTimestamp: String, message: String) {
+        if(threadTimestamp == ""){
```

Comment:
Can we combine the non-thread and thread reply calls to use a single prepared message? We could just optionally set properties depending on if it's a reply to a thread or not.